packages/odyssey-react/src/components/Box/_box-root.scss

```diff
@@ -154,25 +58,5 @@
     max-width: 100%;
   }
```

Comment:
can this style for `svg` be moved into our SvgIcon component? I also think we should avoid styles for `img` unless we have a component for it. Add styles to raw HTML tags from the Box CSS module feels like a weird public API to expose. There should be very few exceptions to this rule (box-sizing border-box `.root * {}` selector for example)